grype/db/vulnerability_provider.go

```diff
@@ -51,6 +51,7 @@ func (pr *VulnerabilityProvider) Get(id, namespace string) ([]vulnerability.Vuln
 	var results []vulnerability.Vulnerability
 	for _, vuln := range vulns {
 		vulnObj, err := vulnerability.NewVulnerability(vuln)
+		vulnObj.PackageName = vuln.PackageName
```

Comment:
I think this just needs to be moved after the error check

Reply:
fixed